src/controllers/user-credential.controller.ts

```diff
@@ -95,42 +93,26 @@ export class UserCredentialController {
         // Add new credential
         const twitterCredential = await this.createCredential(twitterUser);
 
-        return this.transferTips(twitterCredential);
-
-        // const statusTransfer = await this.transferTipsToUser(twitterCredential, twitterUser.id)
-
-        // if(!statusTransfer) throw new HttpErrors.NotFound('RPC Lost Connection')
-
-        // if (twitterUser.id) this.fetchFollowing(twitterUser.id);
+        this.transferTips(twitterCredential);
 
-        // return true
+        return true
 
       case 'reddit':
         const redditUser = await this.reddit(username, publicKey);
-        
-        // Add new credential
         const redditCredential = await this.createCredential(redditUser)
 
-        // this.transferTipsToUser(redditCredential)
-
-        // const statusTransferReddit = await this.transferTipsToUser(redditCredential, 't2_' + redditUser.id)
-
-        // if (!statusTransferReddit) throw new HttpErrors.NotFound('RPC Lost Connection')
+        this.transferTips(redditCredential)
 
-        return this.transferTips(redditCredential)
+        return true
 
       case 'facebook':
```

Comment:
untuk yang hardcode `facebook` mitol tambahin comment `//TODO: move to single constant enum platform`

Reply:
Sip2, sebenarnya yg api terbaru udah gw masukin ke enum